source/guides/maintenance/developer-workstation.rst

```diff
@@ -17,67 +17,204 @@ other Linux\* distributions.
+
+         * - Basic OpenVINO™ toolkit.
+           - `computer-vision-openvino <https://clearlinux.org/software/bundle/computer-vision-openvino/>`_
+
+         * - API helper for cloud access.
+           - `cloud-api <https://clearlinux.org/software/bundle/cloud-api/>`_
+
+         * - All content for pkgconfig file opencv.pc.
+           - `devpkg-opencv <https://clearlinux.org/software/bundle/devpkg-opencv/>`_
+
+   .. tab:: Cloud Orchestration Engineer
+
+      .. list-table:: 
+         :widths: 50, 50
+         :header-rows: 1
+
+         * - Function
+           - Bundles
+
+         * - Contains Clear Linux\* OS native software for cloud.
+           - `ethtool <https://clearlinux.org/software/bundle/ethtool/>`_
+
+         * - Utilities for controlling TCP/IP networking and traffic control.
+           - `iproute2 <https://clearlinux.org/software/bundle/iproute2/>`_
+
+         * - API helper for cloud access.
+           - `cloud-api <https://clearlinux.org/software/bundle/cloud-api/>`_
+
+         * - C++ runtime support.
+           - `libstdcpp <https://clearlinux.org/software/bundle/libstdcpp/>`_
+
+         * - Load and enumerate PKCS#11 modules.
+           - `p11-kit <https://clearlinux.org/software/bundle/p11-kit/>`_
+
+   .. tab:: Game Developer
+
+      .. list-table:: 
+         :widths: 50, 50
+         :header-rows: 1
+
+         * - Function
+           - Bundles
+
+         * - All packages required to build the games bundle.
+           - `games-dev <https://clearlinux.org/software/bundle/games-dev/>`_
+
+         * - Godot game engine editor.
+           - `Godot <https://clearlinux.org/software/flathub/godot/>`_
+
+         * - Real-time game engine (Flatpak).
+           - `Unity Hub <https://clearlinux.org/software/flathub/unity-hub/>`_
+   
+         * - Classic point and click adventure game engine and (Flatpak).
+           - `adventure editor <https://clearlinux.org/software/flathub/adventure-editor/>`_
+
+   .. tab:: Kernel Developer
+
+      .. list-table:: 
+         :widths: 50, 50
+         :header-rows: 1
+
+         * - Function
+           - Bundles
+
+         * - Installs kernel, initrd, kernel config, system map; creates a bootloader entry.
+           - `kernel-install <https://clearlinux.org/software/bundle/kernel-install/>`_
+
+         * - Support module for building/loading via Dynamic Kernel Module System (DKMS) in LTS kernel.
+           - `kernel-lts-dkms <https://clearlinux.org/software/bundle/kernel-lts-dkms/>`_
+
+         * - Support module for building/loading via Dynamic Kernel Module System (DKMS) in native kernel.
+           - `kernel-native-dkms <https://clearlinux.org/software/bundle/kernel-native-dkms/>`_
+
+         * - Support module for building/loading via Dynamic Kernel Module System (DKMS) in AWS kernel.
+           - `kernel-aws-dkms <https://clearlinux.org/software/bundle/kernel-aws-dkms/>`_
+
+         * - Run the Kernel-based Virtual Machine (KVM) specific kernel.
```

Comment:
I'd clarify that this bundle is targeted toward running Clear Linux as a guest under KVM.

Reply:
Thanks.